gosrc/import_path.go

```diff
@@ -118,13 +118,29 @@ func resolveStaticImportPath(importPath string) (*Directory, error) {
 			VCS:         "git",
 		}, nil
 
+	// The next two cases check for dependencies that actually come from the Go
+	// standard library. This approach to resolving the import paths could link
+	// the user to a newer version of a golang/org/x/ package than what is in use
+	// when go-langserver is released. That's acceptable because we don't support
+	// multiple Go versions, we don't publish which Go version our language server
+	// is using anywhere, and these dependencies are very rare in practive (99% of
+	// users are unlikely to encounter them).
 	case strings.HasPrefix(importPath, "golang.org/x/"):
```

Comment:
Apologies, I mean that `golang_org` is from the stdlib. `golang.org/x/` is not, and is widely used everywhere. This comment should be on the below case, not here.